includes/controls/groups/typography.php

```diff
@@ -133,15 +133,15 @@ protected function init_fields() {
 			'type' => Controls_Manager::SELECT,
 			'default' => '',
 			'options' => [
-				'100' => '100',
-				'200' => '200',
-				'300' => '300',
-				'400' => '400',
-				'500' => '500',
-				'600' => '600',
-				'700' => '700',
-				'800' => '800',
-				'900' => '900',
+				'100' => esc_html_x( '100 (Thin)', 'Typography Control', 'elementor' ),
+				'200' => esc_html_x( '200 (Extra Light)', 'Typography Control', 'elementor' ),
+				'300' => esc_html_x( '300 (Light)', 'Typography Control', 'elementor' ),
+				'400' => esc_html_x( '400 (Normal)', 'Typography Control', 'elementor' ),
+				'500' => esc_html_x( '500 (Medium)', 'Typography Control', 'elementor' ),
+				'600' => esc_html_x( '600 (Semi Bold)', 'Typography Control', 'elementor' ),
+				'700' => esc_html_x( '700 (Bold)', 'Typography Control', 'elementor' ),
+				'800' => esc_html_x( '800 (Extra Bold)', 'Typography Control', 'elementor' ),
+				'900' => esc_html_x( '900 (Black)', 'Typography Control', 'elementor' ),
 				'' => esc_html__( 'Default', 'elementor' ),
 				'normal' => esc_html__( 'Normal', 'elementor' ),
 				'bold' => esc_html__( 'Bold', 'elementor' ),
```

Comment:
shouldn't those ones also have `esc_html_x`?

Reply:
Yes they should, but if we replace `esc_html__()` with `esc_html_x()`, we will have to translate those string all over again in [translate.WordPress.org](https://translate.WordPress.org/). A translation string with context is a different translation string then a translation string without context. For now, let's leave as it is. I will create a separate PR for i18n issue.

Reply:
@rami-elementor I think you can change it in this PR, its only 3 strings.

Reply:
@arielk @StyleShit Done.